## Service 
int8 callFlag
---
geometry_msgs/PoseStamped goal
